Detector/DetFCChhTrackerTkLayout/compact/FCChh_v3.03_Definitions.xml

```diff
@@ -3813,19 +2555,24 @@
                 </rings>
             </discZPls>
             <discZPls id="2" z="10985.605*mm" zmin="10976.312*mm" zmax="10994.899*mm" rmin="49.500*mm" rmax="1550.163*mm" bigDelta="5.000*mm">
```

Comment:
Starting from here there is also a duplication of the IDs. Also I do not fully understand how this works, because there is a change in how the discs are declared starting from here. It is done similar as in the inner or outer endcap but the constructor is different (Conical) and in Acts I do not get any modules for these layers.

Reply:
These Id's aren't used anymore, the modules are connected to the detelements that don't exist for repeatedly placed discs. either you get them from the first disc, or I will try cloning the DetElements

Reply:
Why do the DetElements not exist for repeatedly placed discs? Sensitive detector elements need to be unique and should be dd4hep::DetElements.